api_tests/lib/bitcoind_instance.ts

```diff
@@ -41,7 +41,7 @@ export class BitcoindInstance implements LedgerInstance {
         });
 
         this.process.on("exit", (code: number, signal: number) => {
-            console.log(`bitcoind exited with ${code || "signal " + signal}`);
+            console.log(`bitcoind exited with ${code || `signal ${signal}`}`);
```

Comment:
Hum, what is this lint rule?

Reply:
Not using + on mixed types like numbers and strings.